### ISSUE TITLE
Validation failed:  Missing required field: External URL

### ISSUE THREAD

**Issue by joshniesler** [getgrav/grav-plugin-admin#846]
I keep getting "Validation failed: Missing required field: External URL" when trying to select "External URL" as the Display Template for a page.

Bug?

**Comment by rhukster** [comment]
Yes, meant to fix this but forgot about it.  Will ensure it's looked at for next update.

**Comment by aph-vsn** [comment]
Can you please provide a sample path to the external template? Where are the templates suppose to live in the project structure? Thanks!
